src/OpenDB/src/lefin/reader.cpp

```diff
@@ -543,17 +543,16 @@ bool lefin_parse(lefin* lef, utl::Logger* logger, const char* file_name)
   lefrSetDeltaNumberLines(1000000);
   lefrInit();
   int res;
-  if(boost::algorithm::ends_with(file_name, ".gz"))
-  {
+  if (boost::algorithm::ends_with(file_name, ".gz")) {
     auto zfile = lefGZipOpen(file_name, "r");
     if (zfile == NULL) {
-      logger->warn(utl::ODB, 270, "error: Cannot open zipped LEF file {}", file_name);
+      logger->warn(
+          utl::ODB, 270, "error: Cannot open zipped LEF file {}", file_name);
```

Comment:
Why is this a logger warning that prints "error"?

Reply:
That's a good point though in this PR it is just a whitespace change and the problem was pre-existing.